less/mixins/grid.less

```diff
@@ -6,8 +6,8 @@
 .container-fixed(@gutter: @grid-gutter-width) {
   margin-right: auto;
   margin-left: auto;
-  padding-left:  (@gutter / 2);
-  padding-right: (@gutter / 2);
+  padding-left:  floor((@gutter / 2));
```

Comment:
Because .mak-row has a **negative** margin. Therefore rounding must be reversed.
```
margin-left: ceil((15px / -2)); => margin-left: -7px
padding-left: floor((15px / 2)); => padding-left: 7px
```
Sorry for my bad english. :)